lib/ui/blocs/account_read/account_bloc.dart

```diff
@@ -117,6 +117,7 @@ class AccountBloc extends Bloc<AccountEvent, AccountState> {
   Stream<AccountState> _mapGenerateAccountEventToState() async* {
     yield CreatingAccount();
     final mnemonic = await _generateMnemonicUseCase.generate();
+    await Future.delayed(const Duration(seconds: 3));
```

Comment:
Can we change this to be 2 seconds? I think 3 might look a little bit too long

```suggestion
    await Future.delayed(const Duration(seconds: 2));
```